From fed037afbe78b47d46dbbd5838468e57bfe19884 Mon Sep 17 00:00:00 2001
From: Michel Thebeau <michel.thebeau@windriver.com>
Date: Thu, 11 Aug 2016 18:24:25 -0400
Subject: [PATCH] passwd: remove unused default users and groups

A customer request to remove unused users and groups.  These are default
users/groups on centos which have no bearing on a Titanium Cloud cluster.

Signed-off-by: Michel Thebeau <michel.thebeau@windriver.com>
---
 group  | 8 --------
 passwd | 8 --------
 2 files changed, 16 deletions(-)

diff --git a/group b/group
index 825edbb..0a0a1b0 100644
--- a/group
+++ b/group
@@ -1,11 +1,7 @@
 root::0:
-bin::1:
-daemon::2:
 sys::3:
-adm::4:
 tty::5:
 disk::6:
-lp::7:
 mem::8:
 kmem::9:
 wheel::10:
@@ -14,12 +10,8 @@ mail::12:
 man::15:
 dialout::18:
 floppy::19:
-games::20:
 tape::33:
-video::39:
-ftp::50:
 lock::54:
-audio::63:
 nobody::99:
 users::100:
 postgres:x:120:
diff --git a/passwd b/passwd
index 548435f..46a3d52 100644
--- a/passwd
+++ b/passwd
@@ -1,15 +1,7 @@
 root:*:0:0:root:/root:/bin/bash
-bin:*:1:1:bin:/bin:/sbin/nologin
-daemon:*:2:2:daemon:/sbin:/sbin/nologin
-adm:*:3:4:adm:/var/adm:/sbin/nologin
-lp:*:4:7:lp:/var/spool/lpd:/sbin/nologin
 sync:*:5:0:sync:/sbin:/bin/sync
 shutdown:*:6:0:shutdown:/sbin:/sbin/shutdown
 halt:*:7:0:halt:/sbin:/sbin/halt
-mail:*:8:12:mail:/var/spool/mail:/sbin/nologin
-operator:*:11:0:operator:/root:/sbin/nologin
-games:*:12:100:games:/usr/games:/sbin/nologin
-ftp:*:14:50:FTP User:/var/ftp:/sbin/nologin
 nobody:*:99:99:Nobody:/:/sbin/nologin
 rabbitmq:x:121:121::/var/lib/rabbitmq:/bin/sh
 nova:x:994:162::/var/lib/nova:/bin/false
-- 
2.7.4

